agenthub/codeact_agent/codeact_agent.py

```diff
@@ -217,6 +218,16 @@ def _get_messages(self, state: State) -> list[dict[str, str]]:
             {'role': 'user', 'content': self.in_context_example},
         ]
 
+        workspace_contents = ', '.join(list_files(config.workspace_base))
```

Comment:
This may also require mounting workspace to work when running `swe-bench` eval similarly to RepoMap before? CC: @xingyaoww

Reply:
> mounting workspace to work

work means?

Reply:
`codeact_swe_agent.py` is only used for `swe-bench`. right?